tests/suites/sanity/sanity_test.go

```diff
@@ -90,6 +93,55 @@ var _ = Describe(TestName, func() {
 		assertNumberOfCassandraNodes(NodeCount)
 	})
 
+	It("provides metrics to prometheus", func() {
+		podTpl := v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "curl-pod",
+				Namespace: "default",
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "curl",
+						Image:   "curlimages/curl",
+						Command: []string{"sleep", "300"},
+					},
+				},
+			},
+		}
+
+		pod, err := kubernetes.NewPod(Client, podTpl)
+		Expect(err).To(BeNil())
+
+		for {
+			if pod.Status.Phase == v1.PodRunning {
+				break
+			} else {
+				pod, err = kubernetes.GetPod(Client, pod.Name, pod.Namespace)
+				Expect(err).To(BeNil())
+			}
+		}
+
+		var stdout strings.Builder
+		var stderr strings.Builder
+
+		cmd := cmd.New("curl").
+			WithArguments("-s", "prometheus-kubeaddons-prom-prometheus.kubeaddons.svc.cluster.local:9090/api/v1/query?query=cassandra").
```

Comment:
Very nice! Does it also make sense to create a new "module", like `prometheus.go` and encapsulate the usage of the `curl` container? Or at least to extract the Prometheus URL to somewhere else, maybe a constant or function?

Reply:
Yeah, that would probably make sense to abstract into the test-tools

Reply:
+1 for prometheus helpers in `test-tools`!

Reply:
Well, rather curl-helpers. Maybe Prometheus as well...